packages/react/src/components/Dropdown/index.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import ReactSelect from 'react-select';
+import Icon from '../Icon';
+import UiText from '../UiText';
+import { safeInvoke } from '../../utils';
+
+const customStyles = {
+  control: () => ({
+    border: 0,
+    backgroundColor: 'transparent',
+    minHeight: 'initial',
+    display: 'flex',
+    justifyContent: 'space-between',
+  }),
+  indicatorSeparator: provided => ({
+    ...provided,
+    display: 'none',
+  }),
+  indicatorContainer: provided => ({
+    ...provided,
+    marginRight: '1rem',
+    padding: 0,
+  }),
+  valueContainer: provided => ({
+    ...provided,
+    padding: 0,
+    marginLeft: '1rem',
+    minHeight: 38,
+  }),
+};
+
+const CustomDropdownIndicator = ({ innerRef, innerProps }) => (
+  <Icon
+    ref={innerRef}
+    {...innerProps}
+    icon={Icon.ICONS.IconChevronUpDown}
+    className='f-FormEl-icon'
+    size={Icon.SIZES.L}
+  />
+);
+
+const CustomSingleValue = ({ children }) => (
+  <UiText as='span' type={UiText.VARIANTS.content}>
+    {children}
+  </UiText>
+);
+
+const buildCustomOption = (onChange, isSelectable) => ({
+  data,
+  innerProps,
+  isFocused,
+  setValue,
+}) => ({
+  ...data.labelComponent,
+  props: {
+    ...innerProps,
+    ...data.labelComponent.props,
+    focused: isFocused,
+    onClick: e => {
+      safeInvoke(onChange, data.value);
+      safeInvoke(data.labelComponent.props.onClick, e);
+      isSelectable ? innerProps.onClick(e) : setValue(null);
+    },
+  },
+});
+
+const parseListItemObjectsIntoOptionObjects = objects =>
```

Comment:
I don't get what this fn is doing and it's quite verbose 😅

Reply:
Now that I've read it, yeah you're right, it's not that straightforward :)

So what it does:
1. What you're passing to `Dropdown` is an array of `Item`s
2. What you need to pass to `react-select` is a JSON label-value array.
3. So what it does it's wrapping every single `Item` into label-value object

Agree name is verbose. Can't think of anything better though. Have any ideas?

Reply:
`getItems`?